src/common/util.h

```diff
@@ -444,4 +451,41 @@ OSCAP_API char *oscap_realpath(const char *path, char *resolved_path);
  */
 OSCAP_API char *oscap_basename(char *path);
 
+/**
+ * Return directory component of a path
+ * @param path path
+ * @return dirname component of path
```

Comment:
pretty bad docs... do I free what it returns? does it dupe the input or does it reuse it and destroy it?

Reply:
OK